build/Makefile

```diff
@@ -116,6 +116,9 @@ SYSTEST_CONFIG := $(BASE_DIR)/test/config/systest.conf
 .PHONY: tests test omstest unittest systemtest systemtestrb systemtestsh
 
 all : $(SCX_INTERMEDIATE_DIR) $(DSC_TARGET_DIR) $(RUBY_TESTING_DIR) $(RUBY_DEST_DIR) $(IN_PLUGINS_LIB) kit
+        # After a successful make, undo the Ruby patches for sequential builds from old branches
+	@echo "Cleaning patched Ruby files..."
+	cd $(BASE_DIR)/source/ext/ruby; git checkout -- ext/openssl/ossl_ssl.c ext/openssl/extconf.rb
```

Comment:
This patching was already implemented in ruby's tplunk branch, so it might not be needed if we upgrade to a newer version of ruby. can you update the description of this PR , or comment section to mention the version that patching was implemented? The description is very broad.